tests/cypress/support/commands.js

```diff
@@ -50,3 +50,57 @@ from core.models import User
 User.objects.filter(username='${username}').delete()
 "`)
 )
+
+/**
+ * Delete a single comparison made by a given user.
+ */
+Cypress.Commands.add('deleteOneComparisonOfUser', (username, uidA, uidB) => {
+  cy.sql(`
+    DELETE FROM tournesol_comparisoncriteriascore
+    WHERE comparison_id = (
+      SELECT id
+      FROM tournesol_comparison
+      WHERE entity_1_id = (
+        SELECT id FROM tournesol_entity WHERE uid = '${uidA}'
+      ) AND entity_2_id = (
+        SELECT id FROM tournesol_entity WHERE uid = '${uidB}'
+      ) AND user_id = (
+        SELECT id FROM core_user WHERE username = '${username}'
+      )
+    );
+  `);
+
```

Comment:
This delete is required since Django doesn't support database-level cascading (see: https://code.djangoproject.com/ticket/21961)

SQL queries like `DELETE FROM tournesol_comparison` currently fail because of the presence of related items.